addons/binding/org.openhab.binding.avmfritz/src/main/java/org/openhab/binding/avmfritz/handler/DeviceHandler.java

```diff
@@ -179,6 +180,10 @@ public void handleCommand(ChannelUID channelUID, Command command) {
                 if (command instanceof DecimalType) {
                     BigDecimal temperature = new BigDecimal(command.toString());
                     fritzBox.setSetTemp(ain, HeatingModel.fromCelsius(temperature));
+                } else if (command instanceof IncreaseDecreaseType) {
+                    BigDecimal settemp = (BigDecimal) getThing().getConfiguration().get(THING_SETTEMP);
+                    fritzBox.setSetTemp(ain, HeatingModel.fromCelsius(command.equals(IncreaseDecreaseType.INCREASE)
```

Comment:
Should it maybe use the item state for that and maybe use that configuration item only if the value is undefined. Are there other bindings supporting and increase without the remote api doing so and what do they use?